packages/dev/core/src/Culling/ray.ts

```diff
@@ -319,7 +319,7 @@ export class Ray {
      * @param results array to store result in
      * @returns Array of picking infos
      */
-    public intersectsMeshes(meshes: Array<DeepImmutable<AbstractMesh>>, fastCheck?: boolean, results?: Array<PickingInfo>): Array<PickingInfo> {
+    public intersectsMeshes(meshes: DeepImmutable<Array<AbstractMesh>>, fastCheck?: boolean, results?: Array<PickingInfo>): Array<PickingInfo> {

```

Comment:
The idea here is that the meshes are not being changed (i.e. immutable), the array is less of a concern. Is there a typescript-specific reason for that change?